pkg/asset/stock/stock.go

```diff
@@ -70,3 +70,15 @@ func EstablishStock() *Stock {
 	s.metadataStock.EstablishStock(s, s)
 	return s
 }
+
+//
+// func AssetConstructor(typeof string) (Asset, err) {
```

Comment:
??

Reply:
rebase relic :). Some similar code might be if we ever need to be able to construct the asset object from its name. Removed.